merchant-sdk/sdk/src/doc/source/flow.rst

```diff
@@ -1,253 +1,54 @@
-
-.. warning::
-
-    You need to override the ``gms_privacy_policy_link_url`` string resource to provide a link to your company's privacy
-    policy page. This link will be shown to users in the answer to the "Who or what is Gini?" question.
-
-Show the BankSelectionBottomSheet
----------------------------------
-
-The ``BankSelectionBottomSheet`` displays a list of available banks for the user to choose from. If a banking app is not
-installed it will also display its Play Store link.
-
-To instantiate it use ``BankSelectionBottomSheet.newInstance()`` and pass in your ``PaymentComponent`` instance:
-
-.. code-block:: kotlin
-
-    BankSelectionBottomSheet.newInstance(paymentComponent)
-
-
-Show the ReviewFragment
------------------------
-
-The ``ReviewFragment`` displays an invoice's pages and extractions. It also lets users pay the invoice with the bank
-they selected in the ``BankSelectionBottomSheet``.
-
-To instantiate it use ``paymentComponent.getPaymentReviewFragment()`` and pass in the Gini Health API's document id of
-the invoice and the configuration for the screen. Also set a listener to get informed of events from the fragment:
-
-.. code-block:: kotlin
-
-    val reviewConfiguration = ReviewConfiguration(...)
-
-    val paymentReviewFragment = paymentComponent.getPaymentReviewFragment(
-        documentId, reviewConfiguration
-    )
-
-    paymentReviewFragment.listener = object : ReviewFragmentListener {
-        override fun onCloseReview() {
-            // Called only when the ``ReviewConfiguration.showCloseButton`` was set to ``true``.
-            // Dismiss the ReviewFragment.
-        }
-
-        override fun onToTheBankButtonClicked(paymentProviderName: String) {
-            // Log or track the used payment provider name.
-            // No action required, the payment process is handled by the Gini Merchant SDK.
-        }
-    }
-
-.. note::
+    Optionally, you can pass in an instance of ``PaymentFlowConfiguration``.
 
-    ``paymentComponent.getPaymentReviewFragment()`` will load the document extractions asynchronously. It's a suspend
-    function and must be called from a coroutine. 
+    The ``PaymentFlowConfiguration`` class contains the following options:
 
-    The ``ReviewFragment`` doesn't handle navigation related events and doesn't show a navigation bar. You are
-    free to design navigation to and from the fragment as you see fit.
+    - ``shouldHandleErrorsInternally``: If set to ``true``, ``GiniMerchant`` will handle errors internally and show
+      snackbars for errors. If set to ``false``, errors will be ignored by the ``GiniMerchant``. In this case the flows
+      exposed by ``GiniMerchant`` should be observed for errors. Default value is ``true``.
+    - ``showReviewFragment``: If set to ``true``, the user will be able to review the payment details before continuing with the payment. Default value is ``false``.
 
```

Comment:
[Request] Please add another `..note::` to mention that users can still edit the payment information in their banking app even if `showReviewFragment` is set to `false`.